src/LimitedLDLFactorizations.jl

```diff
@@ -244,13 +218,15 @@ function lldl_factorize!(
   wa1 = S.wa1
   wa1 .= zero(Tv)
   s = S.s
-  @inbounds @simd for col = 1:n
+  @inbounds for col = 1:n
```

Comment:
It could help to solve https://github.com/JuliaSmoothOptimizers/LimitedLDLFactorizations.jl/issues/57.